pkg/get/tools.go

```diff
@@ -1134,5 +1134,33 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}_{{.Version}}_{{$osString}}_{{$arch}}{{$ext}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "FairwindsOps",
+			Repo:    "polaris",
+			Name:    "polaris",
+			Version: "3.2.1",
+			BinaryTemplate: `
+				{{$arch := "386"}}
+				{{ if (or (eq .Arch "x86_64") (eq .Arch "amd64")) -}}
+				{{$arch = "amd64"}}
+                		{{- else if eq .Arch "armv6l" -}}
```

Comment:
Do we honestly expect people to run this on Raspberry Pi Zero / RPi 1? If not (I don't) then let's remove it to simplify the logi cand the tests.